test/unit/TokenHolder/getCoGatewayRedeemExecutableData.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Web3 = require('web3');
+const { assert } = require('chai');
+
+const TokenHolder = require('../../../lib/ContractInteract/TokenHolder');
+
+describe('TokenHolder.getCoGatewayRedeemExecutableData()', () => {
+  let tokenHolder;
+
+  beforeEach(() => {
+    const web3 = new Web3();
+    const address = '0x0000000000000000000000000000000000000002';
+    tokenHolder = new TokenHolder(web3, address);
+  });
+
+  it('should construct with correct parameters', async () => {
+    const amount = '100';
+    const beneficiary = '0x0000000000000000000000000000000000000005';
+    const gasPrice = '300';
+    const gasLimit = '10000000';
+    const nonce = '4';
+    const hashLock = '0x0000000000000000000000000000000000000000000000000000000000000002';
+
+    const encodedAbi = tokenHolder.getCoGatewayRedeemExecutableData(
+      amount,
+      beneficiary,
+      gasPrice,
+      gasLimit,
+      nonce,
+      hashLock
+    );
+
+    const expectedEncodedAbi =
```

Comment:
I think it would be helpful to add a brief comment here explaining the choice to go with a fixed value.

Reply:
Will do.